src/java.xml/share/classes/module-info.java

```diff
@@ -221,6 +220,85 @@
  * <tbody>
  *
  * <tr>
+ * <th scope="row" style="font-weight:normal" id="EELimit">entityExpansionLimit</th>
+ * <td>Limits the number of entity expansions.
+ * </td>
+ * <td rowspan="9">yes</td>
+ * <td rowspan="9">yes</td>
+ * <td rowspan="9">Integer</td>
+ * <th id="Value" scope="row" style="font-weight:normal" rowspan="9">
+ * A positive integer. A value less than or equal to 0 indicates no limit.
+ * If the value is not an integer, a NumericFormatException is thrown.
```

Comment:
Is it a ```NumberFormatException```? Also, @link might be useful here.

Reply:
Oh my, the typo has been in the tutorial all this time! It's NumberFormatException indeed.